assets/csidriveroperators/azure-disk/08_deployment.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           value: ${KUBE_RBAC_PROXY_IMAGE}
         - name: CLUSTER_CLOUD_CONTROLLER_MANAGER_OPERATOR_IMAGE
           value: ${CLUSTER_CLOUD_CONTROLLER_MANAGER_OPERATOR_IMAGE}
+        - name: OPERATOR_IMAGE_VERSION
+          value: ${OPERATOR_IMAGE_VERSION}
```

Comment:
This does not get replaced automatically, you still need to provide the replacement around https://github.com/openshift/cluster-storage-operator/blob/d91b59b316111cb434461ff71eeeb96a93912813/pkg/operator/csidriveroperator/csioperatorclient/azure-disk.go#L18-L22

Same for azure-file

Reply:
Ack, done.